p2p/peer.go

```diff
@@ -201,8 +203,9 @@ func (p *Peer) run() (remoteRequested bool, err error) {
 		readErr    = make(chan error, 1)
 		reason     DiscReason // sent to the peer
 	)
-	p.wg.Add(1)
+	p.wg.Add(2)
 	go p.readLoop(readErr)
+	go p.pingLoop()
```

Comment:
Which func call pingLoop() before this change?

Reply:
no func all pingLoop before.